src/main/java/org/betonquest/betonquest/database/PlayerData.java

```diff
@@ -179,6 +183,8 @@ public void removeTag(final String tag) {
         synchronized (tags) {
```

Comment:
I think we want a if there now like in line `166` so the PlayerTagRemoveEvent is only called if the player had the tag before